### PR TITLE
feat(provider/google): Added all cloudrun commands in halyard

### PR DESCRIPTION
**Pull request by sanopsmx**
This is part of https://github.com/spinnaker/governance/issues/302

Added all cloudrun commands in halyard.

halyard cloudrun commands:

1. hal config provider appengine [parameters] [subcommands]
       account: Manage and view Spinnaker configuration for the cloudrun provider’s account
       disable: Set the cloudrun provider as disabled
       edit: Edit cloudrun’s app engine configuration.
       enable: Set the cloudrun provider as enabled

2. hal config provider cloudrun account ACCOUNT [parameters] [subcommands]

     add: Add an account to the cloudrun provider.
     delete: Delete a specific cloudrun account by name.
     edit: Edit an account in the cloudrun provider.
     get: Get the specified account details for the cloudrun provider.
     list: List the account names for the cloudrun provider.
     
3. hal config provider cloudrun account delete ACCOUNT [parameters]

      ACCOUNT: The name of the account to operate on.

4. hal config provider cloudrun account get ACCOUNT [parameters]

      ACCOUNT: The name of the account to operate on.
      
5. hal config provider cloudrun disable [parameters]

6. hal config provider cloudrun account list [parameters]

7. hal config provider cloudrun enable [parameters]
      
   
   
Parameters 
--deployment: If supplied, use this Halyard deployment. This will not create a new deployment.
--no-validate: (Default: false) Skip validation.

### CODE REVIEW

**Comment by sanopsmx** [comment]
Added a video of all the halyard cloudrun commands. It is working as expected. Pls let me know if you have any more issues..

https://github.com/user-attachments/assets/d2759377-ebbe-46f5-a454-9d8522ca6933

**Review by kirangodishala — COMMENTED** [review]
Can you list all the commands in the PR description?

**Comment by sanopsmx** [comment]
> Can you list all the commands in the PR description?

Listed out all commands in the PR description. Pls can you check once

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.netflix.spinnaker.halyard.cli.command.v1.config.providers.cloudrun;
+
+public class CloudrunCommandProperties {
+  static final String LOCAL_REPOSITORY_DIRECTORY_DESCRIPTION =
+      "A local directory to be used to stage source files"
+          + " for Cloud Run deployments within Spinnaker's Clouddriver microservice.";
+  static final String GIT_HTTPS_USERNAME_DESCRIPTION =
```
Comment (kirangodishala) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java:7]: Is it useful to cloudrun? What is the usecase with GIT?

**Review by kirangodishala — APPROVED** [review]

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java (hunk 1 above)
Comment (sanopsmx) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java:7]: It is not useful for cloudrun. we need to remove.

Comment (sanopsmx) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java]: @kirangodishala Can you review once now?

Comment (kirangodishala) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/cloudrun/CloudrunCommandProperties.java]: looks fine.

**Review by kirangodishala — APPROVED** [review]

**Pull request merged**